Feat: remove lorem ipsum wording

Opixelum opened the pull request:
I had an issue with the previous PR (#42)

This PR is the good one. I have translated the wording in french, and replace the lorem ipsum one with it.

n1c01a5 commented on public/locales/fr/homepage.json:
```diff
@@ -15,25 +15,25 @@
 
 	"features_accounting_heading": "Comptabilité",
-	"features_accounting_description": "Lorem ipsum, dolor sit amet consectetur adipisicing elit. Maiores impedit perferendis suscipit eaque, iste dolor cupiditate blanditiis ratione.",
+	"features_accounting_description": "La comptabilité est souvent un calvaire, surtout lorsqu'il s'agit de cryptomonnaies, qui sont souvent instables. Une interface simple calcule l'équivalent en euros et en dollars pour vous simplifier la vie.",
```
`La comptabilité est souvent une tâche pénible, d'autant plus s'il s'agit de cryptomonnaies, qui sont souvent très volatiles. Une interface simple calcule l'équivalent en euros et en dollars pour vous simplifier la vie.`

n1c01a5 commented on public/locales/fr/homepage.json:
```diff
@@ -15,25 +15,25 @@
 	"features_heading": "Des performances en temps réel pour un impact réel",
 	"features_subheading": "Gestion de Projet",
-	"features_description": "Directly on Github your will create the deal and it will automatically appears to developpers avec une preuve de mise sous séquestre de la récompense pour inciter les developeurs à réaliser la tache.",
+	"features_description": "",
```
Directement en créant une Issue sur Github, une mission apparaîtra automatiquement aux développeurs avec une récompense sous séquestre pour encourager les développeurs à accomplir la micro-tâche.

Opixelum commented:
@n1c01a5
- I've updated the accounting description, as asked;
- I've added a better description for the features block;
- I've edited the video demo description, tell me what do you think of it.
